recipe/meta.yaml

```diff
@@ -76,6 +76,8 @@ outputs:
         - {{ compiler('c') }}
         - cmake
       host:
+        # hmaarrfk -- libprotobuf added to ensure that each protobuf builds has a a different hash
+        - libprotobuf
```

Comment:
why do we need to add protobuf for outputs, which don't depend on it? 
I suppose it's OK that libopenvino has the same hash for both builds, because it does not depend on protobuf.

BTW, I will ask again - maybe we can switch back to static version? OpenVINO perfectly works with it.

Reply:
maybe i can change the upload scripts to not fail if the file already exists.

They should work even if the file already exists on the server.

We aren't using `pin_subpackag(exact=True)` so it should be ok.

Let me look into fixing this in conda-smithy.


> BTW, I will ask again - maybe we can switch back to static version? OpenVINO perfectly works with it.

This isn't really an openvino/protobuf only problem.

One main advantage of using shard libraries is that we can distribute ownership of improvements and compilation options.

For example, in something that matters to you, you can help onnxruntime compile with libopenvino, without forcing all the onnxruntime maintainers to understand the details of the compilation options of openvino.


If anypackage should compile statically with libprotobuf it is the offending package that constantly breaks with the shared library, that is, tensorflow


It would be good if you could comment on
https://github.com/conda-forge/tensorflow-feedstock/issues/378 mentionning how tensorflow compatibility (which openvino clearly care about having a "tensorflow" output) is increasing the support burden for the `libopenvino` feedstock.

Reply:
Hopefully we can get it in with: 
https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/306

then just rerun the failed builds.